teckers/teckers/Models/Network/Router/BatchRouter.swift

```diff
@@ -0,0 +1,44 @@
+//
+//  BatchRouter.swift
+//  teckers
+//
+//  Created by Maggie Mendez on 15/11/19.
+//  Copyright © 2019 Teckers. All rights reserved.
+//
+
+import Foundation
```

Comment:
This is just an advice you can create a base Router and then create BatchRouter, ProgramRouter as a child of BaseRouter